lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/mapping/RuleBasedIdentifierMappingUtils.java

```diff
@@ -1,61 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.base.mapping;
-
-import com.google.common.collect.ImmutableList;
-import io.airlift.units.Duration;
-
-import java.nio.file.Path;
-import java.util.List;
-
-import static io.airlift.json.JsonCodec.jsonCodec;
-import static java.nio.charset.StandardCharsets.UTF_8;
-import static java.nio.file.Files.createTempFile;
-import static java.nio.file.Files.write;
-import static java.util.concurrent.TimeUnit.MILLISECONDS;
-
-public final class RuleBasedIdentifierMappingUtils
```

Comment:
ah, and here we go. in `Move RuleBasedIdentifierMappingUtils to module source path` commit, this class shows up as deleted, not moved

(see earlier comment)